exercises/rna-transcription/Example.cs

```diff
@@ -8,8 +8,13 @@ public class Complement
             { 'G', 'C' }, { 'C', 'G' }, { 'T', 'A' }, { 'A', 'U' }
         };
 
-    public static string OfDna(string nucleotide)
+    public static string ToRna(string nucleotide)
```

Comment:
The build fails because this (example) class' name has not been renamed to `RnaTranscription`.